### PR TITLE
Fix overzealous alerts

### PR DESCRIPTION
**Pull request by lekoala**
Regular alert should not be reported as invalid tabs

For the whole story, see #1163

### CODE REVIEW

**Comment by kinglozzer** [comment]
Thanks for investigating this @lekoala - I also use alerts in the same way for displaying help/info to users and had encountered the same issue!

Do we need to check for other “invalid” class names too? `.error`, `.bad` and `.validation` are ones that I’ve previously seen used though I’m not sure if the form validation logic still sometimes adds these.

**Comment by lekoala** [comment]
I assumed the same as the code above. Alert error makes sense since it's bootstrap classes. I wouldn't check for alert bad or anything like that... Maybe message bad is still used somewhere but i believe it's probably better to push a single convention for that system in order to keep it predictable.

**Comment by kinglozzer** [comment]
@lekoala could you run this through the build process?

**Comment by lekoala** [comment]
> @lekoala could you run this through the build process?

Honestly I haven't setup the build process and I'm not sure how long it would take to properly do it. I just made the pull request to have some progress on that issue. It's probably easier for someone that is actively working on the project to run the build process.

**Review by kinglozzer — APPROVED** [review]

**Comment by kinglozzer** [comment]
Ping @Cheddam @emteknetnz, any objections to this?

tldr; using `LiteralField::create('Notice', '<div class="alert info">Info box for CMS user</div>')` results in the tab being highlighted as containing errors after saving, due to the presence of the `.alert` class. This PR changes the code responsible to look for `.alert.error` instead.
